save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/testsuiteselector/TestSuiteSelector.kt

```diff
@@ -54,6 +60,7 @@ enum class TestSuiteSelectorMode {
 @Suppress("TOO_LONG_FUNCTION", "LongMethod")
 fun ChildrenBuilder.showTestSuiteSelectorModal(
     isOpen: Boolean,
+    specificOrganizationName: String?,
     preselectedTestSuiteIds: List<Long>,
```

Comment:
```suggestion
    isOpen: Boolean,
    preselectedTestSuiteIds: List<Long>,
    specificOrganizationName: String? = null,
```

Reply:
Specifying of `specificOrganizationName` changes mode for `TestSuiteSelector`. I think it should be specified explicitly.
But I can add a new method:
```suggestion
fun ChildrenBuilder.showPublicTestSuitesSelectorModal(
    isOpen: Boolean,
    preselectedTestSuiteIds: List<Long>,
    onSubmit: () -> Unit,
    onTestSuiteIdUpdate: (List<Long>) -> Unit,
    onCancel: () -> Unit,
) {
    showTestSuiteSelectorModal(isOpen, null, onSubmit, onTestSuiteIdUpdate, onCancel)
}
```

What do you think?

Reply:
refactored a bit, introduced WindowOpenness which stores state of about window is opened and pass state for selected test suites ids in selector to method

Reply:
Looks nice now